packages/@okta/vuepress-site/docs/reference/api/zones/index.md

```diff
@@ -500,17 +588,17 @@ This operation requires [URL encoding](/docs/reference/api-overview/#filtering).
 
 See [Filtering](/docs/reference/api-overview/#filtering) for more information about the expressions used in filtering.
 
-##### Request Example
+##### Request example
```

Comment:
Would it be helpful to explicitly callout that filters can include only `id` and `usage`?

Reply:
Yep, actually I've mentioned it at https://github.com/okta/okta-developer-docs/pull/1568/files#diff-ffd62a98850737abc7222302c2ed76312428c5e040b7cbc5aa92d02f797fef59R448. Probably worth reiterating here as well.